libs/langchain/langchain/hub.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, Optional
+
+from langchainhub import Client
+
+from langchain.load.dump import dumps
+from langchain.load.load import loads
+from langchain.utils import get_from_dict_or_env, get_from_env
+
+
+def _get_client(kwargs: Dict[str, Any]) -> Client:
+    api_url = get_from_dict_or_env(kwargs, "api_url", "LANGCHAIN_HUB_API_URL")
+
+    # read hub-specific api key
+    api_key = get_from_dict_or_env(kwargs, "api_key", "LANGCHAIN_HUB_API_KEY", None)
+
+    # use default api key as fallback
+    if api_key is None:
+        api_key = get_from_env("api_key", "LANGCHAIN_API_KEY")
+    return Client(api_url, api_key=api_key)
```

Comment:
if you default to None (L14) think it'll still raise an error. also slight pref to explicitly type params

```suggestion
def _get_client(api_url: Optional[str]=None, api_key: Optional[str]=None) -> Client:
    api_url = api_url or get_from_env("api_url", "LANGCHAIN_HUB_API_URL")
    api_key = api_key or get_from_env("api_key", "LANGCHAIN_HUB_API_KEY", default="")
    api_key = api_key or get_from_env("api_key", "LANGCHAIN_API_KEY")
    return Client(api_url, api_key=api_key)
```